packages/x-date-pickers/src/DateTimePicker/DateTimePickerTabs.tsx

```diff
@@ -51,6 +48,11 @@ export interface ExportedDateTimePickerTabsProps extends ExportedBaseTabsProps {
    * @default Time
    */
   timeIcon?: React.ReactNode;
+  /**
+   * Variant of the component this component is inside of.
+   * @default 'desktop'
+   */
+  wrapperVariant?: WrapperVariant;
```

Comment:
Did experiment with making this prop private, but then casting is needed in some places. And I'm not sure if we should stop users from overriding it if they so need. 🤔